README.md

```diff
@@ -379,17 +411,103 @@ Documentation Forthcoming
 
 ### Modules
 
-#### `loadModuleData`
+#### Loading Modules
 
-Documentation Forthcoming
+In both the Server and the Browser, the Holocron Modules loaded depends on which path is matched to a URL using the [child routes configuration](#routing), or when a React component uses the [Holocron API](https://github.com/americanexpress/holocron/blob/master/packages/holocron/API.md).
+
+Both methods are described in the following.
+
+**Contents**
+* React Router Component
+  * [`ModuleRoute`](#moduleroute)
+* Load and Render
+  * [`RenderModule`](#rendermodule)
+  * [`holocronModule`](#holocronmodule)
+  * [`composeModules`](#composemodules)
+
+##### React Router Component
+
+> 👍 Most commonly used method to load Holocron Modules
+
+A parent Module may add the `ModuleRoute` routing component to the [`childRoutes` Module Lifecycle Hook](#routing) to load a child Module dynamically on the server or browser when matching a route path. Once the Module is loaded, it is injected as a JSX element into the `children` prop of the parent Module.
+
+###### `ModuleRoute`
+
+Please see [`ModuleRoute`](https://github.com/americanexpress/holocron/tree/master/packages/holocron-module-route#-usage) in the Holocron Module Route API.
+
+##### Dispatch and Render
+
+We may use the `holocronModule` Higher Order Component to dispatch Holocron Redux Actions. Using the `load` argument in `holocronModule` we dispatch `composeModules` to retrieve a child Module bundle (e.g. `mymodule.browser.js`) and pass React `props` to it. Once loaded, a parent Module may add the `RenderModule` React Component into their JSX to render loaded Holocron Modules.
+
+###### `RenderModule`
+
+Please see [`RenderModule`](https://github.com/americanexpress/holocron/blob/master/packages/holocron/API.md#rendermodule) in the Holocron API.
+
+###### `holocronModule`
+
+Please see [`holocronModule`](https://github.com/americanexpress/holocron/blob/master/packages/holocron/API.md#holocronmodule) in the Holocron API.
+
+###### `composeModules`
+
+Please see [`composeModules`](https://github.com/americanexpress/holocron/blob/master/packages/holocron/API.md#composemodules) in the Holocron API.
+
+#### Loading Data
+
+When [Holocron Modules](#modules) are composed and loaded on the Server and Client, the `loadModuleData` Module Lifecycle Hook is called to load any async requests. The `fetchClient` injected into the `loadModuleData` Hook may be customized using [`createSsrFetch`](#createssrfetch).
```

Comment:
The custom `fetchClient` is only for server side fetching with loadModuleData

Reply:
This is not accurate as we 1) [load `fetchClient` with `enhancedFetch` clientside](https://github.com/americanexpress/one-app/blob/master/src/client/prerender.js#L35) and 2) [`composeModules` pulls `fetchClient` off of Thunk third argument](https://github.com/americanexpress/holocron/blob/master/packages/holocron/src/ducks/compose.js#L22).

Reply:
Fixed in fb577c4